Kitodo-API/src/main/java/org/kitodo/api/dataformat/ProcessingNote.java

```diff
@@ -45,6 +45,33 @@ public String getName() {
         return name;
     }
 
+    /**
```

Comment:
Why do you rearrange getters and setters?
Please group them as a pair

Reply:
Normally git works and does amazing things. But here it has caught the wrong file version on the rebase for some unknown reason. I have undone that.